src/mavsdk/core/mavlink_ftp.cpp

```diff
@@ -216,7 +216,7 @@ void MavlinkFtp::process_mavlink_ftp_message(const mavlink_message_t& msg)
 
 MavlinkFtp::~MavlinkFtp()
 {
-    _system_impl.unregister_all_mavlink_request_message_handlers(this);
+    //_mavlink_request_message_handler.unregister_all_handlers(this);
```

Comment:
```suggestion
    _mavlink_request_message_handler.unregister_all_handlers(this);
```

Reply:
Not sure this is used or where it's from. I'm removing it for now.

Reply:
Not sure where this was used. I'm removing it.